shared/js/background/classes/tab.es6.js

```diff
@@ -31,6 +31,7 @@ const Tracker = require('./tracker.es6')
 const HttpsRedirects = require('./https-redirects.es6')
 const Companies = require('../companies.es6')
 const browserWrapper = require('./../$BROWSER-wrapper.es6')
+const webResourceKeyRegex = new RegExp(/.*\?key=(.*)/)
```

Comment:
nit: any reason this isn't just a literal?

Reply:
I'm using it to grab the key value from the URL https://github.com/duckduckgo/duckduckgo-privacy-extension/pull/550/files#diff-01e7fc66be8e54287b802759209bc548d149d35206f17c8ab3b2e1dbc57cbd6bR162